### DISCUSSION TITLE
Welcome to beanie Discussions!

### DISCUSSION THREAD

**Discussion by roman-right** [BeanieODM/beanie#311] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by grimmerk** [comment]
Hi @roman-right , 
Thank you for supplying this great tool again. I may start to use beanie for my other project(s). I rememeber that I asked the question below in discord and would like to confirm with you again. Is it possible to run beanie object without MongoDB connection? 
Sometimes I want to write the part of the code to parse data first, then sending the data to the DB is the second. In which case, it will be good to use bean objects without MongoDB for quick prototyping/developing/debugging for the first part.
